.npmrc

```diff
@@ -0,0 +1,2 @@
+# Helps with using npm package-locally, while in a yarn workflow (avoids coupling scripts to yarn)
+scripts-prepend-node-path=auto
```

Comment:
This instructs node to prepend the "real"/"correct" node's standard library path to the load path... Instead of it using the default/system one. This removes the risk of your running a node engine version with a library that's from a different version.

`yarn` does "shim out" `node` to ensure that the _same_ runtime is used throughout the process... Which makes builds more reliable & repeatable.